### PR TITLE
Use psalm as phar

### LINKED ISSUE #31: nikic/php-parser v5
My app reported:
```
packages are not up to date:

  - nikic/php-parser (v4.18.0) latest is v5.0.0


brick/varexporter         0.4.0   requires nikic/php-parser (^4.0)          
cakephp/bake              3.0.5   requires nikic/php-parser (^4.13.2)       
phpunit/php-code-coverage 10.1.11 requires nikic/php-parser (^4.18 || ^5.0) 
sebastian/complexity      3.2.0   requires nikic/php-parser (^4.18 || ^5.0) 
sebastian/lines-of-code   2.0.2   requires nikic/php-parser (^4.18 || ^5.0) 
```

### PR DESCRIPTION
**Pull request by snapshotpl**
Partially fixes #31

### CODE REVIEW

**Comment by snapshotpl** [comment]
This PR open way to update php-parser to v5

File: composer.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "nikic/php-parser": "^4.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^8.5 || ^9.0",
+    "phpunit/phpunit": "^9.6.18",
```
Comment (BenMorel) [inline composer.json:14]: `^9.0` is fine here.

**Review by BenMorel — CHANGES_REQUESTED** [review]
Otherwise LGTM @snapshotpl!

File: composer.json (hunk 1 above)
Comment (snapshotpl) [reply composer.json:14]: I don't remembered exactly what, but there was an error when run lower version of phpunit v9

Comment (DonatasIk) [reply composer.json:14]: Could we approve this and move on? Or this needs further investigation? As related packages are waiting for an update

Comment (BenMorel) [reply composer.json]: > I don't remembered exactly what, but there was an error when run lower version of phpunit v9

Indeed, works fine with `^9.3` though, I edited your commit accordingly.

**Comment by BenMorel** [comment]
Thank you, @snapshotpl!

**Pull request merged**